theano/gof/graph.py

```diff
@@ -97,6 +97,7 @@ def __init__(self, op, inputs, outputs):
         self.op = op
         self.inputs = []
         self.tag = utils.scratchpad()
+        utils.add_tag_trace(self)
```

Comment:
I don't think we should put stack trace on the Apply node and Variables. This cause duplicate stack traces. Why did you added it here?

Reply:
I was trying to check multiple places for the print op. Now I removed the other places.